dotnet/iwebapi/k8s/base/route.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: route.openshift.io/v1
+kind: Route
```

Comment:
Switch to ingress, or add route config
https://gitlab.intility.com/DeveloperServices/portal2/my-intility-gitops/-/commit/2097e801cfabe60af801a7b9c95833757720dfe2